src/main/java/org/ibp/api/java/impl/middleware/germplasm/GermplasmNameServiceImpl.java

```diff
@@ -23,33 +31,67 @@
 	@Autowired
 	private GermplasmNameRequestValidator germplasmNameRequestValidator;
 
+	@Autowired
+	private GermplasmCodeNameBatchRequestValidator germplasmCodeNameBatchRequestValidator;
+
 	@Autowired
 	private LocationValidator locationValidator;
 
 	@Autowired
 	private SecurityService securityService;
```

Comment:
Existing in master - `securityService ` is unused. Please cleanup

Reply:
Done